automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/api/orch/OrchestrationApiSpec.scala

```diff
@@ -9,20 +9,19 @@ import org.broadinstitute.dsde.workbench.dao.Google.googleBigQueryDAO
 import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
 import org.broadinstitute.dsde.workbench.service.{Orchestration, RestException}
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
-import org.scalatest.time.{Seconds, Span}
+import org.scalatest.time.{Minute, Seconds, Span}
 import org.scalatest.{FreeSpec, Matchers}
 import org.broadinstitute.dsde.workbench.model.google.GoogleProject
-import org.broadinstitute.dsde.workbench.service.util.Retry
-
-import scala.concurrent.duration._
-import scala.util.Try
 
 class OrchestrationApiSpec extends FreeSpec with Matchers with ScalaFutures with Eventually
   with BillingFixtures {
   implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(5, Seconds)))
 
   "Orchestration" - {
     "should grant and remove google role access" in {
+      // google roles may take a little while to take effect
+      implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(1, Minute)))
```

Comment:
default value for interval is 15 milliseconds - is that too often or are we good with that?

Reply:
That seems excessive.  I can increase.  Say 1 second?